external/uapaotredist/project.json.template

```diff
@@ -0,0 +1,15 @@
+﻿{
+  "frameworks": {
+    "uap10.1": {
```

Comment:
Do you want to make this netcore50 so it is clear we are trying to get those assets?